sources/packages/forms/src/form-definitions/institutionwelcomepage.json

```diff
@@ -481,91 +481,41 @@
                       "components": [
                         {
                           "label": "HTML",
+                          "labelWidth": "",
+                          "labelMargin": "",
                           "tag": "div",
+                          "className": "",
                           "attrs": [
                             {
                               "attr": "",
                               "value": ""
                             }
                           ],
-                          "content": "Step 1",
+                          "content": "<p class=\"category-header-medium primary-color mt-5 mb-5\">Step 1</p>\n<p class=\"category-header-small secondary-color-light mt-5 mb-5\">For admin users</p>\n",
```

Comment:
We can use a <br> here. Does it take more line breaks when you manually use line break to format the code and use <br> as well?

Reply:
what is the issue with <p> tag here?